libs/dh/metering-point/feature-process-overview/src/components/steps.ts

```diff
@@ -94,4 +100,46 @@ export class DhMeteringPointProcessOverviewSteps {
     actor: { accessor: 'actor', cell: (r) => r.actor?.name ?? emDash },
     state: { accessor: 'state' },
   };
+
+  async openRawMessage(documentUrl: string, event: Event) {
+    event.preventDefault();
+    event.stopPropagation();
+
+    try {
+      const content = await firstValueFrom(this.http.get(documentUrl, { responseType: 'text' }));
+
+      if (!content) {
+        this.toastService.open({
+          type: 'danger',
+          message: this.transloco.translate('messageArchive.document.loadFailed'),
+        });
+        return;
+      }
```

Comment:
suggestion: replace this with:

```ts
assertIsDefined(content); // import from @energinet-datahub/dh/shared/util-assert
```

This will update the type if it is defined or throw otherwise. Then you don't need the duplicate toastService call.